<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bth_box" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 222, 222);
border-radius:20px;
</string>
    </property>
    <widget class="QPushButton" name="tools_bth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/menu.png</normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/menu.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="settings_bth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/setting.png</normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/setting.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="home_bth">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/home.png</normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/home.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="notif_bth">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/notification.png</normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/notification.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="test_bth">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/upload.png</normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/upload.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="conn_bth">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/rocket.png</normaloff>../build-qt_proxy-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/icons/rocket.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="connet_lbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(88, 88, 88);</string>
    </property>
    <property name="text">
     <string>connect!</string>
    </property>
   </widget>
   <widget class="QWidget" name="tools_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>351</width>
      <height>501</height>
     </rect>
    </property>
    <widget class="QComboBox" name="choose_proto">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="protocol_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
    <widget class="QLabel" name="country_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="country_line">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="finder_bth">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Find server</string>
     </property>
    </widget>
    <widget class="QLabel" name="browser_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(62, 62, 62);</string>
     </property>
     <property name="text">
      <string>Browse server</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
